litespi/modules.py

```diff
@@ -0,0 +1,12300 @@
+# Generated using 'spi_nor_cfg_gen.py'
+
+class AT25FS010(object):
```

Comment:
Please make all the spi flash modules inherit from a common base module (which is well documented).

Reply:
I added common base module [here](https://github.com/litex-hub/litespi/pull/1/files?file-filters%5B%5D=.patch&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&hide-deleted-files=true#diff-0114c117482b2e8cd170c5a98f7a90d1)